src/AuthorizationUri.php

```diff
@@ -30,6 +32,7 @@ public function __construct(
         private readonly string $clientId,
         private readonly string $redirectUri,
         private readonly string $state,
+        private readonly ?string $nonce = null,
```

Comment:
Musí přijít jako úplně poslední parametr, jinak je to breaking change.

Reply:
ok